netkan/netkan/mirrorer.py

```diff
@@ -0,0 +1,284 @@
+        "CC-BY-NC-ND-2.5"   : 'https://creativecommons.org/licenses/by-nd-nc/2.5/',
+        "CC-BY-NC-ND-3.0"   : 'https://creativecommons.org/licenses/by-nd-nc/3.0/',
+        "CC-BY-NC-ND-4.0"   : 'https://creativecommons.org/licenses/by-nd-nc/4.0/',
+        "CC-BY-ND"          : 'https://creativecommons.org/licenses/by-nd/1.0/',
+        "CC-BY-ND-1.0"      : 'https://creativecommons.org/licenses/by-nd/1.0/',
+        "CC-BY-ND-2.0"      : 'https://creativecommons.org/licenses/by-nd/2.0/',
+        "CC-BY-ND-2.5"      : 'https://creativecommons.org/licenses/by-nd/2.5/',
+        "CC-BY-ND-3.0"      : 'https://creativecommons.org/licenses/by-nd/3.0/',
+        "CC-BY-ND-4.0"      : 'https://creativecommons.org/licenses/by-nd/4.0/',
+        "CC0"               : 'https://creativecommons.org/publicdomain/zero/1.0/',
+        "CDDL"              : 'https://opensource.org/licenses/CDDL-1.0',
+        "CPL"               : 'https://opensource.org/licenses/cpl1.0.php',
+        "EFL-1.0"           : 'https://opensource.org/licenses/ver1_eiffel',
+        "EFL-2.0"           : 'https://opensource.org/licenses/EFL-2.0',
+        "Expat"             : 'https://opensource.org/licenses/MIT',
+        "MIT"               : 'https://opensource.org/licenses/MIT',
+        "GPL-1.0"           : 'http://www.gnu.org/licenses/old-licenses/gpl-1.0.en.html',
+        "GPL-2.0"           : 'http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html',
+        "GPL-3.0"           : 'http://www.gnu.org/licenses/gpl-3.0.en.html',
+        "LGPL-2.0"          : 'https://www.gnu.org/licenses/old-licenses/lgpl-2.0.html',
+        "LGPL-2.1"          : 'https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html',
+        "LGPL-3.0"          : 'http://www.gnu.org/licenses/lgpl-3.0.en.html',
+        "GFDL-1.1"          : 'http://www.gnu.org/licenses/old-licenses/fdl-1.1.en.html',
+        "GFDL-1.2"          : 'http://www.gnu.org/licenses/old-licenses/fdl-1.2.html',
+        "GFDL-1.3"          : 'http://www.gnu.org/licenses/fdl-1.3.en.html',
+        "LPPL-1.0"          : 'https://latex-project.org/lppl/lppl-1-0.html',
+        "LPPL-1.1"          : 'https://latex-project.org/lppl/lppl-1-1.html',
+        "LPPL-1.2"          : 'https://latex-project.org/lppl/lppl-1-2.html',
+        "LPPL-1.3c"         : 'https://latex-project.org/lppl/lppl-1-3c.html',
+        "MPL-1.1"           : 'https://www.mozilla.org/en-US/MPL/1.1/',
+        "Perl"              : 'http://dev.perl.org/licenses/',
+        "Python-2.0"        : 'https://www.python.org/download/releases/2.0/license/',
+        "QPL-1.0"           : 'https://opensource.org/licenses/QPL-1.0',
+        "W3C"               : 'https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document',
+        "Zlib"              : 'http://www.zlib.net/zlib_license.html',
+        "Zope"              : 'http://old.zope.org/Resources/License.1',
+        "Unlicense"         : 'https://unlicense.org/UNLICENSE',
+    }
+
```

Comment:
These are class constants, we could use capitals or an underscore. It might even be worth defining it as a constant though this will be good for now as we're not using it anywhere else.

Reply:
Do you mean renaming it to `LICENSE_URL_DICT`? If so, I'm happy to do that. If not, please clarify how we would use capitals.

Reply:
Apparent confirmation: https://stackoverflow.com/a/45702782/2422988
Will rename them accordingly...